examples/src/bin/embassy_i2s_read.rs

```diff
@@ -66,10 +66,8 @@ async fn main(_spawner: Spawner) {
         &clocks,
     );
 
-    #[cfg(esp32)]
-    {
-        i2s.with_mclk(io.pins.gpio0);
-    }
+    #[cfg(not(features = "esp32"))]
```

Comment:
This `cfg` has changed, was this intentional? Was it incorrect before?

Reply:
Yes, it was intentional because on ESP32 there is only one pin which can be MCLK and it's also rarely used, and also https://github.com/esp-rs/esp-hal/blob/main/esp-hal/src/i2s.rs#L1836C12-L1837